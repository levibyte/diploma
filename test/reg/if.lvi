var var = 9;
print var;
if (a == 8) {
        print true;
} else {
        var i = 1;
        print i;
        print false;
}
print i;
