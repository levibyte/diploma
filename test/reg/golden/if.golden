Expect variable name.
expect expression.
Undefined variable a.
Operands must be number
